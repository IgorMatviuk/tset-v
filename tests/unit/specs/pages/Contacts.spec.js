import { shallowMount } from '@vue/test-utils'

import Contacts from '@/pages/-contacts.vue'

describe('Contacts page', () => {
  test('has expected structure', () => {
    const wrapper = shallowMount(Contacts, {
      mocks: {
        $t: (str) => str,
        $i18n: {
          path: (str) => str
        },
        $store: {
          state: {
            breadcrumb: []
          }
        }
      }
    })

    expect(wrapper.element).toMatchSnapshot()
  })
})
