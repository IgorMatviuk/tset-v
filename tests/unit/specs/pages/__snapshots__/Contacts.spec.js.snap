// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contacts page has expected structure 1`] = `
<main
  class="contacts-page"
>
  <thebreadcrumb-stub
    isfloat="true"
    items="[object Object],[object Object]"
  />
   
  <blockcontacts-stub
    class="contacts-page__block"
  />
</main>
`;
